api/src/test/java/org/openmrs/api/VisitServiceTest.java

```diff
@@ -408,6 +413,20 @@ public void unvoidVisit_shouldUnvoidEncountersVoidedWithVisit() {
 		assertEquals(1, encountersByVisit.size());
 	}
 	
+	@Test
+	public void getVisitsByPatient_shouldReturnEmptyListGivenNull() {
+		assertThat(visitService.getVisitsByPatient(null), is(empty()));
+	}
+	
+	@Test
+	public void getVisitsByPatient_shouldReturnEmptyVisitsForNullPatientId() {
```

Comment:
rename to `getVisitsByPatient_shouldReturnEmptyListGivenPatientWithNullPatientId` to be consistent in naming methods